downstream/modules/platform/con_storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -0,0 +1,14 @@
+[id="con_storage-options-for-operator-installation-on-ocp_{context}"]
```

Comment:
I think we should remove the prefix from the ID for SEO purposes. The ID forms part of the URL and when that includes `con_` etc it is less meaningful. Basically it's just noise to the search engine.